docs/core-concepts.md

```diff
@@ -123,9 +123,57 @@ const [useFirst5SpacedNumbers, first5SpacedNumbers$] = bind(
 
 Something important to note, though, is that the subscription to the shared observable (in this case, `first5SpacedNumbers$`) must have an active subscription before the hook can execute. We can't rely on React renderer to make the initial subscription for us (the subscription which would trigger the side effect), because we can't rely on when rendering happens, nor if it will be interrupted or cancelled.
 
-This means that we need manage the subscription logic for the stream, in order to have full control over when to execute the initial subscription.
+React-RxJS provides different ways of addressing this: The most simple one is to declare the default value for that hook by using the optional argument in `bind`:
+
+```tsx
+import { interval } from "rxjs"
+import { take } from "rxjs/operators"
+import { bind } from "@react-rxjs/core"
+
+const [useFirst5SpacedNumbers, first5SpacedNumbers$] = bind(
+  interval(1000).pipe(take(5)),
+  0 // Default value
+)
+
+function NumberDisplay() {
+  const number = useFirst5SpacedNumbers()
+
+  return <div>{number}</div>;
+}
+
+function App() {
+  return <NumberDisplay />
+}
+```
+
+When a React-RxJS hook has a default value and no one is subscribed to its observable, on the first render it will return that value, and then it will safely subscribe to the source after mounting.
```

Comment:
This is not completely accurate, though... It will return the default value if:
- There are no other subscriptions to the underlying observable or
- If there are other subscriptions, but the underlying observable has not emitted any values, yet.

Otherwise, on the initial render it would yield the latest emitted value (at the time of render).

I'm aware that's not easy to explain this, but I think that we should find a way to explain it, because otherwise a developer may decide against providing the default value because they will be afraid of temporarily rendering a "stale" value.